core/src/main/scripts/importer/validateData.py

```diff
@@ -1192,6 +1181,7 @@ class ClinicalValidator(Validator):
     REQUIRE_COLUMN_ORDER = False
     PROP_IS_PATIENT_ATTRIBUTE = None
     NULL_VALUES = ["[not applicable]", "[not available]", "[pending]", "[discrepancy]","[completed]","[null]", ""]
```

Comment:
In other words, the list on the line referred to here is specific to clinical data files.